modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-9_{context}"]
+= Release notes for {pipelines-title} General Availability 1.9
+
+With this update, {pipelines-title} General Availability (GA) 1.9 is available on {product-title} 4.11 and later versions.
+
+[id="breaking-changes-1-9_{context}"]
+== Breaking changes
+
+// .Pipelines
+
+* With this update, the `Conditions` custom resource definition (CRD) type has been removed. As an alternative, use the WhenExpressions instead.
+
+* With this update, support for `tekton.dev/v1alpha1` API pipeline resources, such as, Pipeline, PipelineRun, Task, Clustertask, and TaskRun has been removed.
+
+* With this update, the `tkn-pac setup` command has been removed. You can now re-use the functionality provided by the `tkn-pac setup` command by using the `tkn-pac webhook add` and `tkn-pac webhook update-token` commands. You can use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. And, you can use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.
+
+[id="deprecated-features-1-9_{context}"]
+== Deprecated and removed features
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, `ClusterTasks` are deprecated and planned to be removed in a future release. As an alternative, you can use `Cluster Resolver`.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the usage of the `triggers` and the `namespaceSelector` fields in a single `EventListener` specification is deprecated and planned to be removed in a future release. You can use these fields in different `EventListener` specifications successfully.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the `tkn pipelinerun describe` command does not display timeouts for the `PipelineRun` resource.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the the `PipelineResource` custom resource (CR) is deprecated. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, custom image parameters in cluster tasks are deprecated. You must run an operator-provided cluster task with a custom image. The following parameters must be configured in the custom image:
+
++
+[source,yaml]
+----
+buildah task doesn't expose "BUILDER_IMAGE" param
+git-clone task doesn't expose "gitInitImage" param
+kn task doesn't expose "kn-image" param
+kn-apply task doesn't expose "KN_IMAGE" param
+maven task doesn't expose "MAVEN_IMAGE" param
+s2i-* tasks don't expose "BUILDER_IMAGE" param
+tkn task doesn't expose "TKN_IMAGE" param
+----
```

Comment:
This section is confusing - those 7 parameters are still in cluster tasks shipped with 1.9 but they will be removed in 1.10.

> You must run an operator-provided cluster task with a custom image
This doesn't make sense. Actually, if a customer wants to use a custom image, they nee d to clone a cluster task and update the image in that newly cloned task.

Reply:
custom image param (for eg: in tnk task `TKN_IMAGE` is a custom image param) from cluster tasks are deprecated.

Reply:
@rupalibehera and @ppitonak - I've made some suggestions as per your comments. Kindly have a look and let me know what you think.